Library/Homebrew/os/linux/elf.rb

```diff
@@ -78,7 +78,7 @@ def with_interpreter?
         begin
           patchelf_patcher.interpreter.present?
         rescue PatchELF::PatchError => e
-          opoo e
+          opoo e unless e.to_s.start_with? "No interpreter found"
```

Comment:
This might be worth an upstream PR: instead of checking the stringified exception here, maybe we should specialize `PatchError` to something like `MissingInterpreterError`.

Reply:
makes sense, that would also clear the exception string matching in the below lines.

Reply:
i.e.:

```ruby
begin
  patchelf_patcher.interpreter.present?
rescue PatchELF::MissingInterpreterError
  false
rescue PatchELF::PatchError => e
  opoo e
  false
end
```